### PR TITLE
Updated Coding schemes to Political Participation

### PR DESCRIPTION
**Pull request by DanielMwendwa**
[Thematic doc](https://docs.google.com/spreadsheets/d/1sA6rAMksRa28zNCfLzf5uHKBSEDMvBWpSnGrib_Hj54/edit#gid=583867014)

### CODE REVIEW

File: code_schemes/rqas/aik/aik_political_participation.json [hunk 1]
```diff
@@ -37,11 +37,115 @@
...
+            "CodeType": "Normal",
+            "DisplayText": "i do not know",
+            "NumericValue": 14,
+            "StringValue": "i_do_not_know",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-5f5aa90d",
+            "CodeType": "Normal",
+            "DisplayText": "not sure",
```
Comment (as2388) [inline code_schemes/rqas/aik/aik_political_participation.json:131]: This is called maybe in the coding frame

File: code_schemes/rqas/aik/aik_political_participation.json [hunk 2]
```diff
@@ -37,11 +37,115 @@
...
+            "CodeType": "Normal",
+            "DisplayText": "no or not interested",
+            "NumericValue": 11,
+            "StringValue": "no_or_not_interested",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-df40be57",
+            "CodeType": "Normal",
+            "DisplayText": "yes, love for politics",
```
Comment (as2388) [inline code_schemes/rqas/aik/aik_political_participation.json:107]: Is it worth sorting these yes, no, don't know?

**Review by as2388 — APPROVED** [review]

File: code_schemes/rqas/aik/aik_political_participation.json (hunk 1 above)
Comment (DanielMwendwa) [reply code_schemes/rqas/aik/aik_political_participation.json]: Removed.

File: code_schemes/rqas/aik/aik_political_participation.json (hunk 2 above)
Comment (DanielMwendwa) [reply code_schemes/rqas/aik/aik_political_participation.json]: Updated, thanks.

**Pull request merged**